code/modules/admin/admin_verbs.dm

```diff
@@ -636,9 +637,12 @@ GLOBAL_LIST_INIT(admin_verbs_maintainer, list(
 
 	log_admin("[key_name(usr)] deadmined themself.")
 	message_admins("[key_name_admin(usr)] deadmined themself.")
+	if(check_rights(R_ADMIN|R_MOD, FALSE))
```

Comment:
We're never going to use R_MOD, but historically it was admins in one group and mods/mentors in the other, iirc they didn't get asay, and they defo showed up in the mentors section of adminwho.